### ISSUE TITLE
[Accessibility] Cookie message close icon does not have an aria-label

### ISSUE THREAD

**Issue by microsoftsam** [Microsoft/pxt#2513]
```
<button tabindex="0" class="ui right floated icon button clear inverted" data-reactid=".0.f.0">
```
![screen shot 2017-07-25 at 4 05 47 pm](https://user-images.githubusercontent.com/16690124/28597515-23e16646-7153-11e7-8c70-cb129d236bf4.png)

**Comment by veler** [comment]
Fixed in [https://github.com/Microsoft/pxt/pull/2533](https://github.com/Microsoft/pxt/pull/2533)
